lib/checks/keyboard/has-at-least-one-main-after.js

```diff
@@ -0,0 +1,16 @@
+var hasMain = false;
+
+//iterate through results from each document
+//stops if any document contains a main landmark
+for (var i = 0; i < results.length && !hasMain; i++) {
```

Comment:
Tip for future work. For-loops generally make code harder to read. JS has a bunch of very excellent array methods which let you write more expressive code. If you had written your code like this, you wouldn't have needed the comment.
```js
const hasMain = results.some(result => result.data === true);
```

Reply:
Great, I'll keep that in mind. Thanks for the tip!